packages/syntax-core/src/Icon/Icon.tsx

```diff
@@ -14,14 +14,20 @@ type IconProps = {
   color?: ComponentProps<typeof Typography>["color"];
   /**
    * The size of the Icon.
-   * * `xs`: 16px x 16px
-   * * `sm`: 20px x 20px
-   * * `md`: 24px x 24px
-   * * `lg`: 32px x 32px
+   * * 100: 16px x 16px
```

Comment:
missing the size 0?

Reply:
Joey didn't spec out a size 0. we can easily add it in the future if that changes